qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/customers/CustomerDatabaseQueries.groovy

```diff
@@ -263,7 +263,7 @@ class CustomerDatabaseQueries {
 
     private static void storeAffiliation(Connection connection, int customerId, List<Affiliation>
         affiliations) {
-        String query = "INSERT INTO customer_affiliation (affiliation_id, customer_id) " +
+        String query = "INSERT INTO person_affiliation (affiliation_id, customer_id) " +
             "VALUES(?, ?)"
 
         affiliations.each {affiliation ->
```

Comment:
I think this class can be removed now.

Reply:
I would rather do that in another PR then